docs/template_plugin/tests/functional/CMakeLists.txt

```diff
@@ -34,4 +34,7 @@ else()
     message("-- Reference preprocessing: OpenCV tests are disabled")
 endif()
 
+# Disable LTO for Template Functional tests
+set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE OFF)
```

Comment:
BTW, I supposed it's not enabled

Reply:
In case that 'lto' fails for templateFuncTests - it is still enabled. However this property didn't help. Now I'm trying to include 'set_input_tensors' tests into existing files - seems like CI may pass.
If so, I'd suggest to merge this PR and raise an issue to improve templateFuncTests overall to fix this 'lto' issue

Reply:
let's try to enable LTO globally for all src/common libraries and remove per target set_target_properties for them

Reply:
ping

Reply:
ping